src/components/events/CheckInFeedback.tsx

```
@@ -0,0 +1,122 @@
+import { createClient } from '@supabase/supabase-js';
```

ESLint

[error] Replace `'`@supabase/supabase-js`'` with `"`@supabase/supabase-js`"`

(prettier/prettier)